src/torchmetrics/classification/__init__.py

```diff
@@ -55,6 +55,8 @@
     MultilabelF1Score,
     MultilabelFBetaScore,
 )
+
+# from torchmetrics.classification.group_fairness import BinaryFairness, BinaryGroupStatRates
```

Comment:
I get the following error when adding these to the init file:

``` bash
python cannot import name 'metric' from partially initialized module 'torchmetrics' 
(most likely due to a circular import)
```

Reply:
Hi @AndresAlgaba, you should import `Metric` as `from torchmetrics.metric import Metric` instead of leveraging `torchmetrics import Metric`. This change should resolve the circular import issue.

Reply:
Thanks for the help! This indeed resolved the issue :)